### ISSUE TITLE
Broken autoloading for use statements with leading \

### ISSUE THREAD

**Issue by e7o-de** [doctrine/annotations#123]
Having issues for some Symfony controllers:

```
<?php

namespace FancyBundle\Controller;

use \Symfony\Bundle\FrameworkBundle\Controller\Controller;
use \Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;

class FancyController extends Controller
{
    /**
     * @Route("/fancy")
     */
    public function fancyAction()
```

When executing php app/console cache:warmup or trying to request a page (using PHP 5.4 in my case, not sure if that matters), I get the following error:

```
// Warming up the cache for the prod environment with debug false                                                      

  [Symfony\Component\Config\Exception\FileLoaderLoadException]                                                                                     
  [Semantical Error] The annotation "@\Sensio\Bundle\FrameworkExtraBundle\Configuration\Route" in method FancyBundle\Controller\FancyController::fancyAction() does not exist, or could not be auto-loaded in /app/fancyProject/current/src/FancyBundle/Controller/ (which is being imported from "/app/fancyProject/current/app/config/routing.yml").
                                                                     
  [Doctrine\Common\Annotations\AnnotationException]               
  [Semantical Error] The annotation "@\Sensio\Bundle\FrameworkExtraBundle\Configuration\Route" in method FancyBundle\Controller\FancyController::fancyAction() does not exist, or could not be auto-loaded.
```

Took me a while, but looks like it only works when the use statement in the controller doesn't contain the leading backslash (fully allowed in PHP), so ... some of that magic happening here is broken.

**Comment by stof** [comment]
duplicate of #115, which is already fixed by #120 (but not yet released)

**Comment by mikeSimonson** [comment]
The new release has a fix for this issue.
